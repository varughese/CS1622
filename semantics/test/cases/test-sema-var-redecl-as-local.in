int foo;

void bar(void)
{
    int foo;
}

void main(void)
{
}
