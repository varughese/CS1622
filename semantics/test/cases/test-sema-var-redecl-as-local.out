[program
  [var-declaration [int] [foo]]
  [fun-declaration
    [void]
    [bar]
    [params]
    [compound-stmt 
      [var-declaration [int] [foo]]
    ]
  ]
  [fun-declaration
    [void]
    [main]
    [params]
    [compound-stmt 
    ]
  ]
]
